### DISCUSSION TITLE
Simple way to refer to parent?

### DISCUSSION THREAD

**Discussion by victormihalache** [tailwindlabs/tailwindcss#7117] [Help] [answered]
I need to change some styling while the parent is being hovered, how would this be done, if possible at all.

This is what the solution looks like now:
```css
.parent:hover > .child {
  ... styling ...
}
```

**Comment by alexanderbluhm** [comment, marked as answer]
Hi @victormihalache
You can use [group ](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state) and group-hover for this.
[Here ](https://play.tailwindcss.com/9zwV9oO9C5) is an example.

**Reply by victormihalache** [reply]
Exactly what i needed, thank you
